docs/contracts.rst

```diff
@@ -194,9 +194,16 @@ return parameter list for functions.
         function setData(uint a) internal { data = a; }
         uint public data;
     }
+    contract Caller {
+        function readData(){
+            C c = new C();
+            uint local = c.data(); // call accessor
```

Comment:
Solution: a link to the Accessor Functions section below OR completely remove the comment?